docs/source/building-supergraphs/jetbrains-ide-support.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Federation support in JetBrains IDEs
+---
+
+The [GraphQL Plugin for JetBrains](https://plugins.jetbrains.com/plugin/8097-graphql/) provides federation-specific development features, such as ___ and ___. **However**, you must _enable_ this federation support after installing the plugin. Otherwise, your IDE might display unexpected errors while you're working with a subgraph schema.
```

Comment:
```suggestion
The [GraphQL Plugin for JetBrains](https://plugins.jetbrains.com/plugin/8097-graphql/) provides federation-specific development features, such as auto-complete for federation directives. **However**, you must _enable_ this federation support after installing the plugin. Otherwise, your IDE might display unexpected errors while you're working with a subgraph schema.
```